lib/pay/braintree/webhooks/subscription_charged_successfully.rb

```diff
@@ -12,7 +12,7 @@ def call(event)
           return unless pay_subscription.present?
 
           billable = pay_subscription.owner
-          charge = billable.save_braintree_transaction(subscription.transactions.first)
+          charge = Pay::Braintree::Billable.new(billable).save_transaction(subscription.transactions.first)
```

Comment:
The main reason I wrote it that way was in case the user had switched payment processors, we'd still process this correctly with Braintree. It's unlikely to happen, but was trying to address edge cases there just to be safe.